tests/Struct/MailingTemplate/MailingTemplatesSourcesTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scn\EvalancheSoapStruct\Struct\MailingTemplate;
+
+use PHPUnit\Framework\TestCase;
+
+class MailingTemplatesSourcesTest extends TestCase
+{
+
+    /** @var MailingTemplatesSources|null */
+    private $subject;
+
+    public function setUp(): void
+    {
+        $this->subject = new MailingTemplatesSources();
+    }
+
+    /**
+     * @dataProvider methodDataProvider
+     */
+    public function testSetterAndGetter(
```

Comment:
Is it possible to move the Method into StructTestCase

Reply:
Sure I can, but this will require a separate testcase